source/user-manual/reference/ossec-conf/global.rst

```diff
@@ -256,17 +256,20 @@ This sets the memory size for the event correlation engine.
 white_list
 ^^^^^^^^^^
 
-This specifies an IP for which Active Responses will not be triggered. Only one IP may be specified for each ``<while_list>`` tag, but several IPs may be used by including multiple ``<white_list>`` tags.
+This specifies an IPv4 address, netblock, or hostname for which Active Responses will not be triggered. Only one of those
+values may be specified for each ``<while_list>`` tag, but several values may be used by including multiple
+``<white_list>`` tags. This configuration is compared against the extracted **srcip** field in the alert.
 
-+--------------------+----------------------------+
-| **Default value**  | n/a                        |
-+--------------------+----------------------------+
-| **Allowed values** | Any IP address or netblock |
-+--------------------+----------------------------+
++--------------------+---------------------------------------------------------------+
+| **Default value**  | n/a                                                           |
++--------------------+---------------------------------------------------------------+
+| **Allowed values** | Any IPv4 address, netblock (i.e.: 192.168.0.0/16) or hostname |
++--------------------+---------------------------------------------------------------+
 
 .. note::
 
-  This option is only valid in server and local installs.
+  This option is only valid in server and local installs. Also, IPv4-mapped IPv6 addresses are allowed too for
+  comparation.
```

Comment:
```suggestion
  This option is only valid in server and local installs.
  IPv4-mapped IPv6 addresses will be compared against any configured IPv4 or netblock.
```

Reply:
Done in [commit](https://github.com/wazuh/wazuh-documentation/pull/3987/commits/395a413280b2ca7ed8a4531e6000a73c1879e6d3)